### ISSUE TITLE
HttpRequestException when trying to run a scenario from Visual Editor

### ISSUE THREAD

**Issue by EinfachToll** [lgsvl/simulator#1858]
Our setup:
- Simulator version: 2021.3
- OS: Ubuntu 20.04.3

Steps:
1. in the Visual Editor, we created a new scenario:
    - map: Straight2LaneSameCurbRightIntersection
    - ego agent: Jaguar2015XE
    - ego agent sensors configuration: Keyboard Control
2. in the browser, we created a new simulation:
    - Runtime Template: Visual scenario editor
    - then we uploaded the JSON file created in step 1

When we try to run this new simulation, it stops immediately and we see the following error:

```
Error on latest run: Test case exception: System.Net.Http.HttpRequestException: Connection failed ---> System.Net.Sockets.SocketException: mono-io-layer-error (10013) at System.Net.Sockets.SocketAsyncResult.CheckIfThrowDelayedException () [0x00014] in <aa976c2104104b7ca9e1785715722c9d>:0 at System.Net.Sockets.Socket.EndConnect (System.IAsyncResult asyncResult) [0x0002c] in <aa976c2104104b7ca9e1785715722c9d>:0 at System.Net.Sockets.SocketTaskExtensions+<>c.<ConnectAsync>b__2_1 (System.IAsyncResult asyncResult) [0x00006] in <aa976c2104104b7ca9e1785715722c9d>:0 at System.Threading.Tasks.TaskFactory`1[TResult].FromAsyncCoreLogic (System.IAsyncResult iar, System.Func`2[T,TResult] endFunction, System.Action`1[T] endAction, System.Threading.Tasks.Task`1[TResult] promise, System.Boolean requiresSynchronization) [0x00019] in <eae584ce26bc40229c1b1aa476bfa589>:0 --- End of stack trace from previous location where exception
```

![Bildschirmfoto vom 2021-11-05 16-56-02](https://user-images.githubusercontent.com/585537/140542000-66664baa-8d37-4656-bf0b-da765cf98522.png)

I'm not sure we did anything suspicious. We can run a simulation with the same map and ego vehicle and the "Random Traffic" template just fine.

**Comment by EricBoiseLGSVL** [comment]
A VSE scenario runner requires bridge and apollo.  Are you using these for the simulation? If not, we are working on supporting VSE without apollo for no bridge or autoware and hope to release soon.

**Comment by EinfachToll** [comment]
We are using the ROS2 bridge, which I omitted here to keep the example simple. The error is the same.

**Comment by EricBoiseLGSVL** [comment]
Yes, ROS2 or no bridge will have the same issue.  We are working on supporting VSE with all/no bridge but currently it needs Apollo/Cyber to run.
